project.yaml

```diff
@@ -938,23 +940,40 @@ Layer:
     <<: *extents
     Datasource:
       <<: *osm2pgsql
+      # This is one of the most complex layers, so it bears explaining in some detail
+      # It is necessary to
+      # - Have roads and railways in the same layer to get ordering right
+      # - Return two linestrings for ways which are both a road and railway
+      # - Compute z_order here, rather than rely on osm2pgsql z_order
       table: |-
```

Comment:
Maybe mention also aeroways? I admit that I am not really sure what is the reason for querying aeroways with roads and railways.

Is it intended to handle cases like [airport/road crossings](http://www.openstreetmap.org/#map=18/36.15107/-5.34812)?